### DISCUSSION TITLE
RFC - Netlify adoption of `gatsby-plugin-netlify`

### DISCUSSION THREAD

**Discussion by lukeocodes** [gatsbyjs/gatsby#31541] [RFC]
Hi folks, I'm working on a project for Netlify, to provide better support for Gatsby features on Netlify's platform.

I believe we can offer better support for Gatsby on Netlify by adopting the `gatsby-plugin-netlify`. Would you be open to us adopting this package fully?

**Comment by wardpeet** [comment]
Hey @lukeocodes,

We're happy to transfer ownership of `gatsby-plugin-netlify` as you'll do a better job maintaining it than us. I would suggest also transferring `gatsby-plugin-netlify-cms`.

We have a shared slack channel. Should we coordinate the move forward in Slack?

**Reply by lukeocodes** [reply]
Sounds like a plan @wardpeet I'll reach out there!
